example/src/components/ArrowBoxExamples.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import ArrowBox from '../../../src/components/ArrowBox';
+import Code from '../../../src/components/Code';
+import Colors from '../../../src/components/Colors';
+
+const directions = ['top', 'right', 'bottom', 'left'];
+
+export default React.createClass({
+  getInitialState() {
+    return {
+      box1Show: false,
+      box2Location: 0,  // 0, 1, 2, 3 ~= 'top', 'right', 'bottom', 'left'
+      box2Show: false,
+    };
+  },
+
+  onClick1() {
+    this.setState({
+      box1Show: !this.state.box1Show,
+    });
+  },
+
+  onClick2() {
```

Comment:
Yea, it feels a bit off to me. I might just suggest having complete code in the first example and omitting the second one. 

If you wanted to go even farther, you could make the example stateless:

```jsx
<div style={{ position: 'relative' }}>
  This is the element that the ArrowBox is anchored to.
  <ArrowBox
    direction="top"
    show={true}
    color={Colors.red_50}
    icon="error"
  >
    <p className="base--p" style={{ color: '#fff', textAlign: 'left' }}>
      This arrow box is now oriented to the <b>top</b>.
    </p>
  </ArrowBox>
</div>
```